common/scala/src/main/scala/whisk/core/entity/Limits.scala

```diff
@@ -72,8 +75,9 @@ protected[core] object ActionLimits extends ArgNormalizer[ActionLimits] with Def
       val time = TimeLimit.serdes.read(obj.get("timeout") getOrElse deserializationError("'timeout' is missing"))
       val memory = MemoryLimit.serdes.read(obj.get("memory") getOrElse deserializationError("'memory' is missing"))
       val logs = obj.get("logs") map { LogLimit.serdes.read(_) } getOrElse LogLimit()
+      val concurrency = obj.get("concurrency") map { ConcurrencyLimit.serdes.read(_) } getOrElse ConcurrencyLimit()
 
-      ActionLimits(time, memory, logs)
+      ActionLimits(time, memory, logs, concurrency)
```

Comment:
is there/will there be a downstream PR to add this to the CLI?

Reply:
indeed; starting with client-go: https://github.com/apache/incubator-openwhisk-client-go/pull/94